client/termdb/tree.js

```diff
@@ -422,7 +389,7 @@ function setRenderers(self) {
 
 		const isSelected = self.state.selectedTerms.find(t => t.name === term.name && t.type === term.type)
 		let text = term.name
-		if (term.isleaf && self.sampleData[term.id]) text += ` (${self.sampleData[term.id]})`
+		if (term.isleaf && self.sampleByTermId[term.id]) text += ` (${self.sampleByTermId[term.id]})`
```

Comment:
@airenzp should use optional chaining here, `sampleByTermId` may not be present.

Also rename `sampleByTermId` to `sampleDataByTermId`.